Remove outdated qualifiers on resources folders

MGaetan89 opened the pull request:
Some resources were still in the `-v21`/`-v22`/`-v23` resources folders.
This PR moves them to the main folder, updating the original files when needed.

MGaetan89 commented:
Good point! I'll look into removing the PNG files when there is a vector equivalent

tonihei commented:
> shouldn't this also remove .png files for example in ui library

Yes, it should, thanks for noting! These pngs only existed to support API<21 devices, so can now be removed entirely.

MGaetan89 commented:
Done.

I've removed PNG files that already have a vector equivalent in https://github.com/androidx/media/pull/2685/commits/a0d7ba3cae95748c4bba123949a7691d679626e6, and converted `exo_icon_circular_play` to vector in https://github.com/androidx/media/pull/2685/commits/9f6154f43a45f3d1d935653e0caeb660a267cb13.

There's still `exo_icon_vr.png` that is not a vector, but I didn't find the icon on https://fonts.google.com/icons

rohitjoins commented:
I'm going to send this for internal review now. You may see some more commits being added as I make changes in response to review feedback. Please refrain from pushing any more substantive changes as it will complicate the internal review - thanks!

MGaetan89 commented:
@rohitjoins with e42940c71885aa98b95f57c1434241dccea7056c, the content of https://github.com/androidx/media/blob/main/libraries/session/src/main/res/values-v21/styles.xml is no longer applied. Is it ok?
If it is, maybe the [`values-v24/styles.xml`](https://github.com/androidx/media/blob/main/libraries/session/src/main/res/values-v24/styles.xml) file can be removed too? Its content is similar to the one in [`values/styles.xml`](https://github.com/androidx/media/blob/main/libraries/session/src/main/res/values/styles.xml) now.

rohitjoins commented:
@MGaetan89 yes, that change is intentional. This is because the values are now provided by the base `androidx.media`.

> If it is, maybe the [values-v24/styles.xml](https://github.com/androidx/media/blob/main/libraries/session/src/main/res/values-v24/styles.xml) file can be removed too?

That's a great observation. Will remove that file too! Thanks!

oceanjules commented:
> There's still `exo_icon_vr.png` in `:lib-ui` that is not a vector, but I didn't find the icon on https://fonts.google.com/icons

How about 
* [head_mounted_device ](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:head_mounted_device:FILL@0;wght@0;GRAD@0;opsz@24&icon.size=24&icon.color=%235f6368) most matches the glasses we have
* [vrpano](https://fonts.google.com/icons?selected=Material+Icons+Outlined:vrpano:&icon.size=24&icon.color=%235f6368&icon.query=realit&icon.set=Material+Icons) doesn't really look like VR

rohitjoins commented:
Had to rollback this change due to internal Gradle build failure, will merge it again after we can find the cause.

MGaetan89 commented:
Thanks for the info. Let me know if I can help in fixing the issue.
